getting-started/templates/systemlink-values.yaml

```diff
@@ -723,6 +723,16 @@ dataframeservice:
     # <ATTENTION> This must be set to the region of the S3 instance.
     ##
     region: *s3Region
+  ## Configure nessie.
+  ##
+  nessie:
+    auth:
+      ## Name of the secret containing the nessie credentials
+      ##
+      secretName: nidataframe-nessie-credentials
```

Comment:
I wouldn't include this value in this template unless you expect the user to need to change it.

Reply:
It's not for an external connection like Mongo or S3. But if the user sets `secrets.deploySecrets` to `false`, they will need to supply the secret. It's the same situation as `dataframeservice.sldremio.auth` below.